sidebars.ts

```diff
@@ -1848,6 +1848,15 @@ integrations: [
         collapsed: true,
         link: {type: 'doc', id: 'integrations/databases/index'},
         items: [
+          {
+            type: 'category',
+            label: 'OpenTelemetry',
+            collapsible: true,
+            collapsed: true,
+            items: [
+              'integrations/databases/opentelemetry/opentelemetry-couchbase',
+            ],
```

Comment:
This is going to cause a merge conflict with the [Redis PR](https://github.com/SumoLogic/sumologic-documentation/pull/1771/files) because it'll consider it two folders named "OpenTelemetry". I recommend instead making one PR _just_ for the sidebars.

@nmohantyctr @sumoanema @angadrandhawa1 @arunpatyal @stacykor